.github/CODEOWNERS

```
@@ -43,32 +58,19 @@
 /examples/gno.land/r/sys/             @moul
 /examples/gno.land/r/jaekwon/         @jaekwon
 /examples/gno.land/r/manfred/         @moul
+/examples/gno.land/r/morgan/          @thehowl
```

can you add 
`/examples/gno.land/r/leon/` for me please? so I don't have to make a separate PR
```suggestion
/examples/gno.land/r/morgan/          @thehowl
/examples/gno.land/r/leon/                @leohhhn
```